aws/utility.sh

```diff
@@ -951,7 +951,16 @@ function deleteTreeFromBucket() {
   local optional_arguments=("$@")
 
   # Delete everything below the prefix
-  aws --region "${region}" s3 rm "${optional_arguments[@]}" --recursive "s3://${bucket}/${prefix}/"
+  aws --region "${region}" s3 rm "${optional_arguments[@]}" --recursive "s3://${bucket}/${prefix}${prefix:+/}"
```

Comment:
Will that put the prefix twice?

Reply:
No. It is using the alternate value operator to add a slash after the prefix if the prefix value is defined and not empty. The format has been used in other routines in this file.